packages/common/src/form-template/form-template.js

```diff
@@ -96,26 +96,16 @@ FormControls.propTypes = {
   buttonsProps: PropTypes.object,
 };
 
-FormControls.defaultProps = {
-  submitLabel: 'Submit',
-  cancelLabel: 'Cancel',
-  resetLabel: 'Reset',
-  canReset: false,
-  canSubmit: false,
```

Comment:
is this not used?

Reply:
It's been moved to the `FormControls` argument defaults.